src/config/metric-names.ts

```diff
@@ -17,25 +17,27 @@ export const queueMetrics = {
 	waiting: `${server}.queue.waiting`,
 	paused: `${server}.queue.paused`,
 	repeatable: `${server}.queue.repeatable`,
+	jobDuration: `${server}.queue.job-duration`,
 }
 
 export const metricHttpRequest = (metricName?: string) => {
 	return {
-		github: `${server}.http.request.github`,
-		jira: `${server}.http.request.jira`,
-		install: `${server}.http.request.install`,
-		uninstall: `${server}.http.request.uninstall`,
-		fullSync: `${server}.http.request.full-sync`,
-		syncPullRequest: `${server}.http.request.sync-pull-request`,
+		duration: `${server}.http.server.requestes.duration`, //Duration of incoming HTTP requests
+		executed: `${server}.http.server.requestes`, //Count of executed incoming HTTP requests
+		github: `${server}.http.request.github`, //Octokit requests execution time histogram
+		jira: `${server}.http.request.jira`, //Jira requests execution time histogram
+		install: `${server}.http.request.install`, //Count how many installation requests we received
+		uninstall: `${server}.http.request.uninstall`, //Count how many uninstallation requests we received
+		syncPullRequest: `${server}.http.request.sync-pull-request`, //Measures retrieval time from GitHub API
 		hydroSubmission: `${server}.http.request.${metricName}`,
-		jobDuration: `${server}.http.request.job-duration`,
```

Comment:
Changed some of these metrics because they have little to do with http requests. (Job duration counts the duration of the queue jobs)

Reply:
I'll open a pull request for our dashboards as well